go/vt/sqlparser/parse_test.go

```diff
@@ -4932,6 +4944,12 @@ var (
 	}, {
 		input:  "insert into a select * into @a from b",
 		output: "INTO clause is not allowed at position 38 near 'b'",
+	}, {
```

Comment:
Another good test case to add would be `SRID` appearing twice in a column definition. That would give us coverage over the code path that checks for that error.